package.json

```diff
@@ -74,7 +74,8 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "resolutions": {
-    "**/@types/node": "10.12.12"
+    "**/@types/node": "10.12.12",
+    "**/typescript": "^3.3.3333"
```

Comment:
LGTM with this, but it's safer to have @spalger take a look on this.

Reply:
👍 until we have something like greenkeeper on the Kibana repo this is a fine way to handle this, but I'd prefer if this was sent to `master` and not the `feature/merge-code` branch.

Reply:
https://github.com/elastic/kibana/pull/32440